src/components/UI/Accordion/AccordionItem.tsx

```diff
@@ -20,23 +23,25 @@ export const AccordionItem: FunctionComponent<AccordionItemProps> = ({
   headingTag = "h5",
   heading,
   divider = false,
+  isOpen = false,
```

Comment:
ps just saw, should be `-1` yeah?

Reply:
oh ya right

Reply:
also if we change this to number means we are moving the check into the component and that we need another const to store the boolean value that is being used everywhere in the component, so if u think it will be better for readablity then i will update, what do u think?